src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -3675,7 +3675,6 @@ def build_bicep_file(cmd, file, stdout=None, outdir=None, outfile=None, no_resto
 
 
 def format_bicep_file(cmd, file, stdout=None, outdir=None, outfile=None, newline=None, indent_kind=None, indent_size=None, insert_final_newline=None):
-    ensure_bicep_installation()
```

Comment:
May I ask why you need to remove this line?

Reply:
[This PR](https://github.com/Azure/azure-cli/pull/25033) introduced changes allowing `bicep` to be picked up from the path based on a config value or the CLI managed version is used, installed if not present. So, this method is being called [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/resource/_bicep.py#L85) eventually and hence this can be removed here.

Reply:
We may need to restore the line, because it has a call to `bicep_version_greater_than_or_equal_to` below before calling `run_bicep_command`. Currently, `bicep_version_greater_than_or_equal_to` doesn't trigger auto installation of Bicep CLI.

Reply:
@JanuszNowak Any update?

Reply:
Restore and added "ensure_bicep_installation(cmd.cli_ctx)"